app/dashboard/static/js/app/view-builds-job-branch-kernel.2020.2.1.js

```diff
@@ -691,19 +690,19 @@ require([
                     .getElementById('unknown-btn').removeAttribute('disabled');
             }
 
-            setTimeout(function() {
-                if (!loadSavedSession()) {
-                    if (hasFailed) {
-                        showFailedOnly();
-                    } else {
-                        html.addClass(
-                            document.getElementById('all-btn'), 'active');
-                    }
+            if (!loadSavedSession()) {
+                console.log("no saved session");
```

Comment:
`if (hasFailed) {
    showFailedOnly();
} else if (!loadSavedSession()) {
}`

What do you think about this update? So, we can avoid problems related to session data.

Reply:
Well if the saved session is to show only passing builds then it should be used, even if there are some failed builds.  The save session will be specific to the kernel revision so the visitor will have already seen the failed builds the first time, before choosing another button.

Reply:
alright!